mitiq/rem/tests/test_inverse_confusion_matrix.py

```diff
@@ -64,20 +70,20 @@ def test_bitstrings_to_probability_vector():
     assert (pv == np.array([0, 0, 0, 1])).all()
 
 
-def test_probability_vector_roundtrip():
-    for _ in range(10):
-        pv = np.random.rand(4)
-        pv /= np.sum(pv)
-        assert isclose(
-            np.linalg.norm(
-                pv
-                - bitstrings_to_probability_vector(
-                    sample_probability_vector(pv, 1000)
-                )
-            ),
-            0,
-            abs_tol=0.1,
-        )
+@pytest.mark.parametrize("_", range(10))
+def test_probability_vector_roundtrip(_):
```

Comment:
micro-nit: what I meant in my previous comment is actually
 ```suggestion
@pytest.mark.parametrize("repeats", range(10))
def test_probability_vector_roundtrip(_):
```

Reply:
Ahh yeah that would be a little more clear, I think. Unfortunately, that is not possible. The string argument to [`pytest.mark.parametrize`](https://docs.pytest.org/en/6.2.x/parametrize.html) has to match the argument names (up to some formatting), so having a completely different name like `repeats` and `_` do not work.
```
============================================================== short test summary info ==============================================================
ERROR mitiq/rem/tests/test_inverse_confusion_matrix.py - Failed: In test_probability_vector_roundtrip: function uses no argument 'repeats'
================================================================= 1 error in 3.29s ==================================================================
ERROR: found no collectors for /Users/nate/code/mitiq/mitiq/rem/tests/test_inverse_confusion_matrix.py::test_probability_vector_roundtrip
```

Reply:
oh, well, thanks for trying